.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-pnpm lint && pnpm format && pnpm test:api
\ No newline at end of file
+npx lint-staged && pnpm test:api
```

Comment:
Since we are using pnpm, i believe `pnpx` would be a better thing to use in here.

Reply:
Why have we removed pnpm format from here

Reply:
We can use task concurrency ig @rajdip-b  
https://www.npmjs.com/package/lint-staged#task-concurrency